eve_glue/notification_type.py

```diff
@@ -282,5 +285,14 @@ class NotificationTypeEnumV2(enum.Enum):
         "CorporationGoalCreated": 260,
         "CorporationGoalClosed": 261,
         "CorporationGoalCompleted": 262,
-    }
+    },
+)
+
+NotificationTypeEnumV10 = new_from_enum(
```

Comment:
should it not be `NotificationTypeEnumV11` ? v10 is already declared at line 277 (280 after these changes)